playbook/app/pb_kits/playbook/packs/site_styles/_samples_index.scss

```diff
@@ -28,15 +29,12 @@
     box-shadow: $shadow_deep;
   }
   // Mobile Styles
-  @include break_at(699px){
-    .full_page_samples_flex_wrapper {
-      flex-direction: column;
-      align-items: center;
-    }
+  @include break_at(731px){
```

Comment:
interesting px choice. Any particular reason for `731`? Just curious!

Reply:
Yeah right?  That's the exact width at which stuff displays wonky if it's not vertically aligned (`flex-direction: column`).  Had to reconfigure it after adding new `min-width` value to the list kit.